pkg/controller/certificates/certificate_controller.go

```diff
@@ -148,7 +153,11 @@ func (cc *CertificateController) processNextWorkItem() bool {
 func (cc *CertificateController) enqueueCertificateRequest(obj interface{}) {
 	key, err := controller.KeyFunc(obj)
 	if err != nil {
-		utilruntime.HandleError(fmt.Errorf("Couldn't get key for object %+v: %v", obj, err))
```

Comment:
what was making us see this a lot? I wouldn't have expected the keyfunc to ever fail

Reply:
wasn't the spammy log message the `utilruntime.HandleError(fmt.Errorf("Sync %v failed with : %v", cKey, err))` line?